Neos.Media/Classes/Domain/Service/AssetService.php

```diff
@@ -321,6 +323,14 @@ public function replaceAssetResource(AssetInterface $asset, PersistentResource $
                             $adjustment->refit($asset);
                         }
                     }
+
+                    if ($redirectHandlerEnabled) {
```

Comment:
Instead of adding this block again, I'd rather move the one from the if-block above to below the if-else. I'll update this PR…

Reply:
That would mean, that a redirect is also added, when methods in the try-catch block fail. Would that be ok?

Reply:
No, the catch does `continue` now.